WordPress/gutenberg
# [RNMobile] Allow to display radial-gradient

## Description

That PR introduces the possibility to display `radial-gradient` in components which are currently supporting gradients on mobile, such as `Buttons` and `Cover`.

ref to gutenberg mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2266

## How has this been tested?

1. Open draft on web
2. Add `Buttons` or `Cover`
3. Set radial gradient background color
4. Save draft
5. Open post on mobile
6. Expect to see radial gradient

## Screenshots 

Screenshots are presenting the same block and gradient color but different type:

* ios
<details>
<img width="482" alt="Screenshot 2020-05-15 at 15 59 22" src="https://user-images.githubusercontent.com/22746080/82058746-74c4ce00-96c5-11ea-855f-e9be675349d0.png">
<img width="482" alt="Screenshot 2020-05-15 at 15 59 29" src="https://user-images.githubusercontent.com/22746080/82058754-768e9180-96c5-11ea-9403-059b51a38abb.png">
</details>

* android
<details>
<img width="482" src="https://user-images.githubusercontent.com/22746080/82059274-164c1f80-96c6-11ea-8236-c933754a0183.png"/>
<img width="482" src="https://user-images.githubusercontent.com/22746080/82059277-1815e300-96c6-11ea-99d2-1afc8969377f.png" />
<img width="482" src="https://user-images.githubusercontent.com/22746080/82059279-19471000-96c6-11ea-96dc-9bad5000cd3b.png" />
</details>

## Types of changes

Use `RadialGradient` from `react-native-svg` to display radial gradient.

## Checklist:
